test/TestExpectations.json

```diff
@@ -2111,6 +2111,12 @@
     "parameters": ["firefox", "webDriverBiDi"],
     "expectations": ["FAIL"]
   },
+  {
+    "testIdPattern": "[elementhandle.spec] ElementHandle specs ElementHandle.isIntersectingViewport should work with svg elements",
```

Comment:
why does this fail?

Reply:
Not sure, but from the stack trace, I suspect there is a bug in Firefox with their code related to `DOM.resolveNode` (because that's where the issue was coming from).

Reply:
I mean why it fails because of this PR? there should be no changes related to that, right?

Reply:
It changes this test because we use migrate this to isolated world. I can revert this change.

Reply:
Ah, that is fine then. I didn't see that isIntersectingViewport was also changed.